lib/routes/account.js

```diff
@@ -1743,21 +1780,59 @@ module.exports = (
         // failed right away.
         request.emitMetricsEvent('email.verify_code.clicked')
 
-        db.account(uid)
-          .then(
-            (account) => {
-              // This endpoint is not authenticated, so we need to look up
-              // the target email address before we can check it with customs.
-              return customs.check(request, account.email, 'recoveryEmailVerifyCode')
-                .then(
-                  () => account
-                )
-            }
-          )
-          .then(
-            (account) => {
+        return db.account(uid)
```

Comment:
is it possible to document the flow here for the future? 

* Customs check
* If given code type is `secondary` then try verify a secondary email
* Otherwise verify the account, this can be done using the sign up code or with a one time code generated by sign-in confirmation.